YouTubeSearch/webpack.dev.js

```diff
@@ -0,0 +1,41 @@
+const path = require('path');
+const { merge } = require('webpack-merge');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
+const common = require('./webpack.common');
+
+module.exports = merge(common, {
+  mode: 'development',
+  output: {
```

Comment:
```suggestion
  output: {
     publicPath: '/',
```
это исправит get error net::ERR_ABORTED